recipes/glib/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# don't get locally installed pkg-config entries:
+export PKG_CONFIG_LIBDIR="${PREFIX}/lib/pkgconfig:${PREFIX}/share/pkgconfig"
```

Comment:
Interesting that this doesn't happen normally. Could we try removing to see if this is still necessary?